.circleci/config.yml

```diff
@@ -0,0 +1,111 @@
+version: 2.1
+jobs:
+  node:
+    working_directory: ~/build
+    docker:
+      - image: cimg/python:3.8.13-node
```

Comment:
So from this website https://circleci.com/developer/images/image/cimg/python?utm_source=google&utm_medium=sem&utm_campaign=sem-google-dg--uscan-en-dsa-tROAS-auth-nb&utm_term=g_-_c__dsa_&utm_content=&gclid=Cj0KCQjw7aqkBhDPARIsAKGa0oIU-5l_QLZWPNB8ebxzbo_O15ynoo8FemKN7WDctz9Nu-vBFKvFmx0aAuEGEALw_wcB 
3.8.13-node has node 16.14.2 
while: https://github.com/crowdbotics/modules/blob/2a211186280210a9dabb104ce9ed021dacec0da7/scaffold/template/custom/.circleci/generate_mobile_android_config.sh#L18 has node 18.16.0

which version of node should we be using. Do the other versions, 
```
build-essential 12.8ubuntu1.1, curl 7.68.0, docker 20.10.12, docker-compose Docker Compose version v2.2.3 v2.2.3, dockerize v0.6.1, git 2.35.1, jq 1.6, node 16.14.2, pip 22.0.4, poetry 1.1.13, pyenv 2.2.5, python2 2.7.18, python3 3.8.13, ubuntu 20.04.3 LTS, virtualenv 20.13.4, wget 1.20.3, wheel 0.37.1, yarn 1.22.5
```

Also match what we are building on the mobile platform?

Reply:
18.16.0. We upgraded our scaffold to use this for our node jobs.

Reply:
That's correct @qcorporation. I had issues installing Python to the `cimg/node:18.16.0 ` image so I found this quick solution. I didn't want to take additional time for this and I chose to use `cimg/python:3.8.13-node`. I also thought of creating a new image but that would also take a bit more time. The least required version for node in the scaffold is `16.0` so I assumed this was good enough for now.

Reply:
@GaboGomezT do you want to talk about the Python issues when using this image? Perhaps we can debug it a bit.

Reply:
Fair enough, I'll work on updating the node version to 18.6.0